ProjectBank/Client/Shared/FormComponent.razor

```diff
@@ -0,0 +1,57 @@
+<EditForm Model="@Project" OnValidSubmt="@HandleValidSubmit">
+    <DataAnnotationsValidator />
+    <ValidationSummary />
+
+    <div class="mb-3">
+        <label class="form-label">Title</label>
+        <InputText id="title" class="form-control" @bind-Value="Project.Title" />
+    </div>
+
+    <div class="mb-3">
+        <label class="form-label">Description</label>
+            <InputTextArea id="description" class="form-control" @bind-Value="Project.Description" />
+    </div>
+
+    <div class="mb-3">
+        <label class="form-label">Tags</label>
+            <InputText id="tags" class="form-control" @bind-Value="@TagListString"/>
+    </div>
+
+    <button type="submit" class="btn btn-success" onclick="@HandleValidSubmit">@ButtonName</button>
+</EditForm> 
+
+@code {
+    [Parameter]
+    public Project Project {get; set;}
```

Comment:
Shouldn't we use the CreateProjectDTO and UpdateProjectDTO instead Project?

Reply:
Probably, I am not sure how this is going to work with EF core, so if this is what is being sent to the repository then yes it would seem that way, so should i change it to that?

Reply:
UpdateProjectDTO already extends CreateProjectDTO :)